lib/Rex/Service.pm

```diff
@@ -61,6 +61,8 @@ sub get {
 
     # this also counts for Ubuntu
     $class = "Rex::Service::Debian::systemd";
+  }elsif ( is_archlinux($operatingsystem) ) {
```

Comment:
Watch out on merge! I'm pretty sure this line is not tidy. I'm not sure yet why `perltidy` and/or the tidy tests missed this one.